database/migrations/2021_04_21_102707_change_statuses_to_accept_business_checkin_as_integer.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+class ChangeStatusesToAcceptBusinessCheckinAsInteger extends Migration
+{
+    public function up() {
+        Schema::table('statuses', function(Blueprint $table) {
+            $table->integer('business')
+                  ->default(0)
+                  ->change();
+        });
+
+        DB::table('statuses')
+            ->whereNull('business')
+            ->update(['business' => '0']);
+        DB::table('statuses')
+            ->where('business', 'true')
+            ->update(['business' => '1']);
+        DB::table('statuses')
+            ->where('business', 'false')
+            ->update(['business' => '0']);
+    }
+
+    public function down() {
+        Schema::table('statuses', function(Blueprint $table) {
+            $table->boolean('business')
+                ->change();
+        });
+
+        DB::table('statuses')
+            ->where('business', '>=', '1')
+            ->update(['business' => true]);
+        DB::table('statuses')
+            ->where('business', '0')
+            ->update(['business' => false]);
```

Comment:
```suggestion
```
See the last comment for this...

Reply:
We can't remove this, since the business-column is defined as a boolean, so we need to store everything greater than 1 as a 1. It's debatable whether we need to define it as true and false, b/c sqlite and afaik MariaDB define it as normal integers.

Reply:
Ah, I see! Then the first instruction makes sense. However, I would still delete the second one.

Reply:
Now it should be better.

Reply:
Thanks!